deployments/upgrading-chocolatey-for-business-clients/README.md

```diff
@@ -0,0 +1,59 @@
+# Upgrading Chocolatey for Business Client Components
+
+## Purpose
+
+These Deployment Plans are examples of upgrading the Chocolatey for Business client components.
+
+There are two example deployments here. They follow the instructions laid out on the [upgrading to Chocolatey v2](https://docs.chocolatey.org/en-us/guides/upgrading-to-chocolatey-v2-v6#upgrade-using-chocolatey-central-management-deployments) docs page.
+
+The first upgrades the Chocolatey for Business client components to the latest 1.* version.
+
+https://github.com/chocolatey/ccm-deployment-examples/blob/b79340df0363be0fcda3192828cef96095320d3d/deployments/upgrading-chocolatey-for-business-clients/Deployment-Latest1x.json#L1-L34
+
+The second upgrades Chocolatey to the latest version.
+
+https://github.com/chocolatey/ccm-deployment-examples/blob/b79340df0363be0fcda3192828cef96095320d3d/deployments/upgrading-chocolatey-for-business-clients/Deployment-Latest2x.json#L1-L34
+
+## Usage
+
+You should import both the `Latest1x` and `Latest2x` deployments if you are on a version of Chocolatey older than 1.4.0, and deploy `Upgrade Chocolatey for Business Client to Latest 1.*` first, before deploying `Upgrade Chocolatey for Business Client` to upgrade to the latest version.
+
+If you are upgraded to Chocolatey CLI version 2.0 or above, you can just import and run `Latest2x`.
+
+> Although the ScheduledTasks module is available on Windows Server 2012 R2, the chocolatey-agent service encounters an error when trying to import it. It is recommended to explore other options for the scheduled task if you're using Windows Server 2012 R2.
```

Comment:
```suggestion
> Although the ScheduledTasks module is available on Windows Server 2012 R2, the `chocolatey-agent` service encounters an error when trying to import it. It is recommended to explore other options for the scheduled task if you're using Windows Server 2012 R2.
```

Reply:
What other options?

Reply:
Jokingly: Upgrading to a more recent OS.

In more seriousness: We don't currently have company recommendations for any alternatives to this. I thought it prudent to include the warning, as I found it interesting when reading the docs, but would rather remove the warning than add recommendations for an SKU that had (initial) extended support end today (🎉). If we really want, I can investigate and create a `DeploymentPlan-2012R2Only.json` with an alternative? I think I have a few options available...